### DISCUSSION TITLE
Recorded macro doesn't execute key mapping

### DISCUSSION THREAD

**Discussion by Drllap** [neovim/neovim#24040] [Q&A]
I have this key mapping in my `init.vim`:
```
inoremap <silent> jk <esc>
```
to exit insert mode with jk.
I recorded a macro to change this, starting from the beginning of `tags`:
```
tags:
  - ubuntu
  - wbmsgen2
```
to this:
```
tags:
  - linux
  - shell
```
the results were: `j2wcwlinuxjkjciwshelljk`.
Now when I move to the next `tags` in the file I get:
```
tags:
  - linux
  - wbmsgen2
```
I suspect that this has something to do with the interaction between macros and the keybinding. This doesn't happen when I run with `nvim --clean --nopulugin` so I'm guessing I have set some option that changes the interaction? Does anyone have any ideas what or how I can find out?

**Comment by TheLeoP** [comment]
How are you executing the macro? Do you have `@` mapped to something else?

**Reply by Drllap** [reply]
I recorded it with `qq` the execute with `@q`

**Reply by TheLeoP** [reply]
Do you have @ mapped to something else? You can check it by `:verbose map @`

**Reply by Drllap** [reply]
no, I get `No mapping found`

**Comment by Drllap** [comment]
I have no idea why, but I can no longer reproduce this
